test/extended/workspacetype/workspace.go

```diff
@@ -0,0 +1,45 @@
+package workspacetype
+
+import (
+	"strings"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[sig-workspace]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		k = exutil.NewCLIWithoutNamespace("kcp-workspace")
+	)
+
+	g.It("Author:pewang-Medium-[Smoke] Multi levels workspaces lifecycle should works well", func() {
```

Comment:
@Phaow would you mind explaning what you are trying to achieve here ? It is a little confusing here with ServerURL, orgServerURL, ParentServerURL or it would be good if we could add defintions to these some where, thanks !!

Reply:
Sure. Maybe it's for current solution for parallelly ececute our test cases when https://github.com/kcp-dev/kcp/issues/1689 finished we could make it simply and just use the `--kubeconfig` instead of `--server`.